src/azure-cli-core/azure/cli/core/commands/parameters.py

```diff
@@ -238,7 +239,10 @@ def _type(value):
     completer=get_resource_group_completion_list,
     id_part='resource_group',
     help="Name of resource group. You can configure the default group using `az configure --defaults group=<name>`",
-    configured_default='group')
+    configured_default='group',
+    lc_actions=[STORE, USE],
```

Comment:
this should impact almost all commands. Is it by design?

Reply:
If existing users are using config right now, I think they have to run the command once again.

Reply:
yes, all resource group defined with this type will support local context. It should not impact az configure. There values are saved in separate file and local context have higher priority than `az configure` if it is turned on.